CMakeLists.txt

```diff
@@ -303,23 +303,31 @@ add_definitions(${LFS_DEFINITIONS})
 add_compile_options(${LFS_COMPILE_OPTIONS})
 target_link_libraries(libgerbera ${LFS_LIBRARIES})
 
-find_package (pupnp "1.12.1" REQUIRED)
+pkg_check_modules (NPUPNP libnpupnp)
```

Comment:
I prefer an explicit switch between pupnp / npupnp, for example in case one has both installed.
Also it would simnplify selection / conditional compilation logic.
If they were are 100% compatible then no changes should be needed, right?